src/x86/intel_edison_fab_c.c

```diff
@@ -414,6 +441,7 @@ mraa_intel_edison_pwm_init_pre(int pin)
 mraa_result_t
 mraa_intel_edison_pwm_init_post(mraa_pwm_context pwm)
 {
+    memset(pwm_disabled, 0, sizeof(pwm_disabled));
```

Comment:
Why is all pwm status being disabled on init? Surely only the pwm disabled flag for the pin in question should be disabled?